remix.init/index.js

```diff
@@ -200,9 +208,23 @@ async function main({ rootDirectory, ...rest }) {
 
   debug(`Start creating Remix app with name`, appName, `in`, rootDirectory)
 
+  // Check if user wants to add authentication
+  const answer = await inquirer.prompt({
+    name: 'provider',
+    message: 'Add Google as authentication provider?',
+    type: 'confirm',
+    default: true,
+  })
+
+  let identityProviders
+  if (answer.provider) {
+    identityProviders = await setupEasyAuth({ url: 'https://todo.replace.me' })
```

Comment:
@maxpaj We have a "moment 22" here. We want the url of the page to add to google app when creating provider credentials, but we do not get it until we've run `setupAzureResources()`, and we can't run `setupAzureResource()` until we have the credentials in place. Any thoughts? Run the setupAzureResource with dummy id and secret and later add it? Maybe rerun `azure provision` after we've got the credentials from user?